pkg/util/mount/mount_linux_test.go

```diff
@@ -1435,6 +1475,32 @@ func TestSafeOpen(t *testing.T) {
 			"test",
 			true,
 		},
+		{
+			"mounting-unix-socket",
+			func(base string) error {
+				testSocketFile := filepath.Join(base, "mount_test.sock")
```

Comment:
do you need to clean this up on exit?

Reply:
Same as below, since we are removing the base directory via https://github.com/kubernetes/kubernetes/blob/master/pkg/util/mount/mount_linux_test.go#L658 , it should remove any files created in the tests.